hera_mc/mc_session.py

```diff
@@ -4764,25 +4856,35 @@ def update_metric_desc(self, metric, desc):
             MetricList.metric == metric)[0].desc = desc
         self.commit()
 
-    def get_metric_desc(self, metric=None):
+    def get_metric_desc(self, metric=None, write_to_file=False, filename=None):
         """
         Get metric description(s) from the M&C database.
 
         Parameters
         ----------
-        metric : str or list of strings
+        metric : str
             Metric name. Defaults to returning all metrics.
+        write_to_file : bool
+            Option to write records to a CSV file.
+        filename : str
+            Name of file to write to. If not provided, defaults to a file in the
+            current directory named based on the table name.
+            Ignored if write_to_file is False.
 
         Returns
         -------
         list of MetricList objects
 
         """
-        args = []
+        query = self.query(MetricList)
+
         if metric is not None:
-            args.append(MetricList.metric.in_(get_iterable(metric)))
+            query = query.filter(MetricList.metric == metric)
```

Comment:
Any particular reason this is changed to only allow a single metric return? I feel like this multiple entry method _could_ be useful, though I bet it was hardly used.

Reply:
This was done in the name of unifying all the `get` methods on MCSession. We talked about it in a telecon and decided to make the metric get methods behave the same as other methods which do not accept lists only single filter values for each column.

Reply:
Ah fair enough, I must have missed that conversation.!